tools/testbuild.sh

```diff
@@ -62,6 +66,7 @@ function showusage {
   echo "  -a <appsdir> provides the relative path to the apps/ directory.  Default ../apps"
   echo "  -t <topdir> provides the absolute path to top nuttx/ directory.  Default $PWD/../nuttx"
   echo "  -p only print the list of configs without running any builds"
+  echo "  -A store the build executable artifact in ARTIFACTDIR (defaults to $PWD/../buildartifacts"
```

Comment:
let's remove $PWD here and line 67 since no other place use it in the whole script

Reply:
I came make it relative to topdir. Does that work?